DESIGN.md

```diff
@@ -171,13 +171,6 @@ You get a flowable from:
 class Flowable<T> implements Flow.Publisher<T>, io.reactivestreams.Publisher<T> {
   void subscribe(Subscriber<T> subscriber);
 
-  interface Subscriber<T> implements Flow.Subscriber<T>, io.reactivestreams.Subscriber<T> {
-    void onNext(T t);
-    void onError(Throwable t);
-    void onComplete();
-    void onSubscribe(Flowable.Subscription d);
-  }
-
```

Comment:
The document no longer defines `Flowable.Subscriber` anywhere. Is that because it is intended to just be the ReactiveStreams `Subscriber` so isn't being specifically called out? If so, perhaps the fully qualified class name should be used to be explicit about that?